manifests/platform/solaris.pp

```diff
@@ -13,11 +13,11 @@
 # Requires: nothing
 #
 class splunk::platform::solaris inherits splunk::virtual {
-  include ::staging
+  include ::archive
   include ::splunk::params
   include ::splunk::platform::posix
 
-  $path         = $staging::path
+  $path         = $archive::path
```

Comment:
There's no such variable as $archive::path

Reply:
Hey @alexjfisher It does work but i might be accessing this the wrong way from the puppet archive module

![image](https://user-images.githubusercontent.com/6224270/30427662-39c67baa-9916-11e7-8c8a-7e02db478211.png)

Reply:
My mistake.  I missed the `inherits` https://github.com/voxpupuli/puppet-archive/blob/master/manifests/init.pp#L28

Reply:
The staging module created $staging::path for you.  With archive, I think you'll need to `include ::archive::staging` instead of just `::archive` ??

Reply:
Done.

Reply:
Also needed in `forwarder.pp` ?

Reply:
Done